<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="rahulshettyacademy.TestComponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="Submitorder(failed)">
    <classes>
      <class name="rahulshettyacademy.Tests.StandAloneTesting">
        <methods>
          <include name="orderHistoryTest"/>
          <include name="teardown"/>
          <include name="lunchAplication"/>
          <include name="SubmitOrder" invocation-numbers="1"/>
        </methods>
      </class> <!-- rahulshettyacademy.Tests.StandAloneTesting -->
    </classes>
  </test> <!-- Submitorder(failed) -->
  <test thread-count="5" parallel="tests" name="ErrorValidation(failed)">
    <classes>
      <class name="rahulshettyacademy.Tests.ErrorValidations">
        <methods>
          <include name="teardown"/>
          <include name="lunchAplication"/>
          <include name="LoginErrorMessage"/>
        </methods>
      </class> <!-- rahulshettyacademy.Tests.ErrorValidations -->
    </classes>
  </test> <!-- ErrorValidation(failed) -->
</suite> <!-- Failed suite [Suite] -->
